base/operators.jl

```diff
@@ -328,9 +328,31 @@ eltype(::Type{Any}) = Any
 eltype(t::DataType) = eltype(supertype(t))
 eltype(x) = eltype(typeof(x))
 
-# function pipelining
+# function pipelining, composition & negation
+
 |>(x, f) = f(x)
 
+"""
+    ∘(f, g)
+    f ∘ g
```

Comment:
At least it shows you that operators are just standard functions (OK, not true for `<:`) and that you can add a method using the standard syntax. But we should definitely improve these docstrings by adding the infix forms on the second line at the minimum.